riker-macros/src/lib.rs

```diff
@@ -4,15 +4,15 @@ use proc_macro2::{Ident, TokenStream};
 use quote::quote;
 use syn::parse::{Parse, ParseStream, Result};
 use syn::punctuated::Punctuated;
-use syn::{DeriveInput, Generics};
+use syn::{DeriveInput, Generics, PathSegment};
```

Comment:
PathSegment should be replaced with TypePath. Otherwise you can't use fully qualified type names.

For example:
```rust
mod some_module {
    #[derive(Clone, Debug)]
    pub struct Message<T> {
        pub inner: T,
    }
}

#[actor(some_module::Message<String>)]
#[derive(Clone, Default)]
struct GenericMsgActor;
```